### PR TITLE
Add tinyalign recipe

### PR DESCRIPTION
**Pull request by marcelm**
This adds a recipe for the Python "tinyalign" package (of which I am the author).
----

> Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences**
      (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

> Everyone has access to the following BiocondaBot commands, which can be given in a comment:
>
>  * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
>  * `@BiocondaBot please add label` will add the `please review & merge` label.
>  * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.
>
> For members of the Bioconda project, the following command is also available:
>
>  * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.
>
> If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

**Comment by marcelm** [comment]
@BiocondaBot please add label

File: recipes/tinyalign/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+test:
+  imports:
+    - tinyalign
+
+about:
+  home: https://github.com/marcelm/tinyalign/
+  license: MIT License
+  license_file: LICENSE
+  summary: 'Banded edit distance'
```
Comment (mbargull) [inline recipes/tinyalign/meta.yaml]: Taken (and shortened) from https://github.com/marcelm/tinyalign/blame/v0.2/README.md#L5-L7:
```suggestion
  summary: A small Python module providing edit distance and Hamming distance computation.
```
If you want to, you can add a detailed description too.

File: recipes/tinyalign/meta.yaml [hunk 2]
```diff
@@ -5,7 +5,7 @@ package:
   version: {{ version }}
 
 source:
-  url: https://files.pythonhosted.org/packages/20/2c/a5c5f356e3d491941fe29aba4d71078e90eb4ea83b26af616b1ac9502ab7/tinyalign-0.2.tar.gz
+  url: https://pypi.io/packages/source/t/tinyalign/tinyalign-{{ version }}.tar.gz
```
Comment (mbargull) [inline recipes/tinyalign/meta.yaml]: This is just so we can be a little more lazy and only update `version` and `sha256` on updates.

Comment (marcelm) [reply recipes/tinyalign/meta.yaml]: Ah, perfect! I didn’t know these short URLs existed and was always annoyed that `{{ version }}` couldn’t be used with the long ones.

**Comment by mbargull** [comment]
@BiocondaBot please fetch artifacts

**Comment by mbargull** [comment]
@BiocondaBot please merge

**Comment by marcelm** [comment]
Thanks!

**Comment by mbargull** [comment]
Likewise, as always!

**Pull request merged**